src/main/java/net/sf/jabref/gui/importer/FetcherPreviewDialog.java

```diff
@@ -302,4 +303,10 @@ public void showMessage(String message, String title, int msgType) {
     public void showMessage(String message) {
         JOptionPane.showMessageDialog(this, message);
     }
+
+    public void showErrorMessage(EntryFetcher fetcher) {
```

Comment:
I change the interface to `public void showFetchingErrorMessage(String fetcherName)` since you don't need the whole fetcher object and the name is more specific for the fetchers. When creating public utility methods it's also good to have a javadoc comment describing it's functionality as it should also be used by others in the future. Other than that I'm fine with your changes 👍